docs/quickstart.rst

```diff
@@ -0,0 +1,83 @@
+============
```

Comment:
Remove the extra =

Reply:
Like the current change to make it the length of the word? (my rst is a little rusty :smile:)

Reply:
I didn't see this but that was what I was referring to thanks